test/unit/BToken.tree

```diff
@@ -0,0 +1,30 @@
+BToken::constructor
+└── when called
+    ├── it sets token name
+    └── it sets token symbol
+
+BToken::increaseApproval
+├── when sender is address zero
+│   └── it should revert
+├── when spender is address zero
```

Comment:
how could we get to this? I imagine the assertion was added to OZ to avoid programeer errors when calling the internal method, but is there any context in where an external caller could be`address(0)`?

Reply:
hmmm is not, but is an actual possible branch, it actually happens inside `_approve` so tbh i wouldn't care so much for it